package.json

```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "private": true,
   "workspaces": [
+    "lition",
```

Comment:
Please also add `lition` to `.github/workflows/release.yml/publish-artifacts` so it's built automatically on every release.